src/Makefile.in

```diff
@@ -119,7 +119,7 @@ PAXCTL_dumped = @PAXCTL_dumped@
 PAXCTL_notdumped = @PAXCTL_notdumped@
 
 ## Some systems define this to request special libraries.
-LIBS_SYSTEM=@LIBS_SYSTEM@
+LIBS_SYSTEM=-L../rust_src/target/debug -lremacs -ldl
```

Comment:
Okay! Do you want me to squish those changes into one commit, or add the new commit for these fixes to this pull request (can you even do that? I haven't really used github's pull requests), or something else?